### DISCUSSION TITLE
Ender 6 BLTouch Position Display

### DISCUSSION THREAD

**Discussion by fuzelet** [mriscoc/Ender3V2S1#465] [General]
Hello,
I've been using your firmware for a few months on my "Ender 3 S1 Pro F4 with a Ender 3 S1 Encoder Screen" and decided to give this firmware and same screen a go on my Ender 6.  I have so far been able to compile, flash and even print with the firmware, however I am running into an issue with the positions being displayed on the bottom of the screen.  The X and Y positions being reported on the main screen are obviously not correct.  It does not appear to effect the ability to print, move, etc but really drives me crazy with them not being accurate (z appears to be accurate).  I have attached my config files as well as two pictures showing the issue.  As you can see from the image, the X and Y positions do not align with being in the middle of a 260 x 260 bed.   Any help would be appreciated. 

Ender 6 uses a X_MAX, Y_MAX and Z_MIN plug.

![PXL_20221025_134731606](https://user-images.githubusercontent.com/70331495/197805441-42821258-e7ef-4957-bf26-f4c695602102.jpg)

![PXL_20221025_134728332](https://user-images.githubusercontent.com/70331495/197805452-b7839d59-e599-45d4-b535-fefe908be619.jpg)

[config.zip](https://github.com/mriscoc/Ender3V2S1/files/9861278/config.zip)

**Comment by fuzelet** [comment]
Just wanted to throw this out there, but maybe the issue is with how coordinates and COREXY machines work?  I as soon as I move either the Y or X axis, the other coordinate will change as well. I am guessing this happening cause the other motor is rotating even though the plane is never changed on the opposite axis?  I can confirm the "Move Axis" menu shows the correct location of the nozzle, the bottom coordinates seem to display whatever they want?

**Auto Home:**
![PXL_20221025_172605470](https://user-images.githubusercontent.com/70331495/197842078-7d23c57c-378c-4024-aac8-87809cc8d150.jpg)


**Move Z to Home:**
![PXL_20221025_172634595](https://user-images.githubusercontent.com/70331495/197842082-32d93521-1862-4d89-8a23-91873274cc2a.jpg)

**Comment by mriscoc** [comment]
Hi, in the dwin.cpp file, change `#define SHOW_REAL_POS 1` to `#define SHOW_REAL_POS 0`

Show real position uses motor step position, in a coreXY printer, that information could give incorrect axis position information.

**Reply by fuzelet** [reply]
Thank you very much for the reply and assistance. Once I changed that variable it seems to be working as intended now. Hope you have a great day!
